distribution/test/itests/test-itests-alliance/src/test/java/alliance/test/itests/TestNsili.java

```diff
@@ -115,49 +119,73 @@ public void beforeAllianceTest() throws Exception {
         }
     }
 
+    /**
+     * Determine if the Nsili Source has been configured in Alliance and available
```

Comment:
We may want to specify that this Nsili Source is HTTP, since two are configured in the itests